components/org.wso2.carbon.appmgt.rest.api.util/src/main/resources/publisher-api.json

```diff
@@ -2772,7 +2771,16 @@
             "in": "path",
             "description": "Stat Type.\n",
```

Comment:
Could you please mention the allowed types in description (refer lifecycle change api as an exp)